src/app/components/client/ExposureCard.module.scss

```diff
@@ -53,6 +53,14 @@
         display: none;
       }
 
+      .exposureCompanyTitle {
```

Comment:
Q: what is this? A HIBP breach `Title`? Or the name of a data reseller?
I know the font is "-sm", but curious I have some mild concerns yet about truncating at 120px (give or take a few px for an &hellip;).

If this is HIBP data, we can probably pretty quickly determine the longest/average `breach.Title` length. If it's exposures coming from OneRep, I don't have API keys or experience to know how to try and get some sample data for that yet.

Reply:
@pdehaan there are some limited examples in https://docs.onerep.com/#operation/getScanResult, on Heroku you should be able to do a scan with the `/redesign/user/welcome` onboarding flow and JSON with full results will be available at `/redesign/user/welcome/results` (this screen will be removed soon, it's just there for debugging until we have the dashboard fully built out).